test/test-moveTetromino.js

```diff
@@ -0,0 +1,829 @@
+        let shift = { x: -1, y: 0 };
+        let lTetromino = new tetris.Tetromino(typeOfTetromino, phase, coordinatesTetromino);
+        let resultMoveTetromino = lTetromino.moveTetromino(field, shift);
+        let expected = tetris.getCoordinates(`
+            -----
+            ----X
+            --XXX
+            -----
+        `)
+    })
+    it("L tetromino can not move, the element is out of the field", function () {
+        let typeOfTetromino = "L";
+        let phase = "left";
+        let coordinatesTetromino = tetris.getCoordinates(`
+            -----
+            --XXX
+            --X--
+            -----
+        `)
+        let field = tetris.buildField(`
+            -----
+            -----
+            -----
+            -----
+        `);
+        let shift = { x: 1, y: 0 };
+        let lTetromino = new tetris.Tetromino(typeOfTetromino, phase, coordinatesTetromino);
+        let resultMoveTetromino = lTetromino.moveTetromino(field, shift);
+        let expected = tetris.getCoordinates(`
+            -----
+            --XXX
+            --X--
+            -----
+        `);
+    })
+    it("L tetromino can not move, item too low", function () {
+        let typeOfTetromino = "L";
+        let phase = "up";
+        let coordinatesTetromino = tetris.getCoordinates(`
+            -----
+            --XX-
+            ---X-
+            ---X-
+        `)
+        let field = tetris.buildField(`
+            -----
+            -----
+            -----
+            -----
+        `);
+        let shift = { x: 0, y: -1 };
+        let lTetromino = new tetris.Tetromino(typeOfTetromino, phase, coordinatesTetromino);
+        let resultMoveTetromino = lTetromino.moveTetromino(field, shift);
+        let expected = tetris.getCoordinates(`
+            -----
+            --XX-
+            ---X-
+            ---X-
+        `);
+    })
+    it("Move down J tetromino", function () {
+        let typeOfTetromino = "J";
+        let phase = "up";
+        let coordinatesTetromino = tetris.getCoordinates(`
+            -XX--
+            -X---
+            -X---
+            -----
+        `)
+        let field = tetris.buildField(`
+            -----
+            -----
+            -----
+            -----
+        `);
+        let shift = { x: 0, y: -1 };
+        let jTetromino = new tetris.Tetromino(typeOfTetromino, phase, coordinatesTetromino);
+        let resultMoveTetromino = jTetromino.moveTetromino(field, shift);
+        let expected = tetris.getCoordinates(`
+            -----
+            -XX--
+            -X---
+            -X---
+        `)
+    })
+    it("J tetromino can not move, there is an element on the way", function () {
+        let typeOfTetromino = "J";
+        let phase = "up";
+        let coordinatesTetromino = tetris.getCoordinates(`
+            -XX--
+            -X---
+            -X---
+            -----
+        `)
+        let field = tetris.buildField(`
+            -----
+            --X--
```

Comment:
this field is not in the way of moving up - test name does not reflect what you are checking